src/main/java/com/aws/greengrass/util/NucleusPaths.java

```diff
@@ -24,9 +27,10 @@ public class NucleusPaths {
     private Path kernelAltsPath;
     private Path cliIpcInfoPath;
     private Path binPath;
+    private Path logsPath;
```

Comment:
we own the logger, the logger has methods to read the path it is going to write to already.